spec/classes/python_spec.rb

```diff
@@ -90,6 +90,13 @@
         it { is_expected.to contain_package("python-dev").with_ensure('absent') }
       end
     end
+
+    describe "EPEL need not apply" do
+      context "default/empty" do
```

Comment:
need to set the facts with values for osfamily and operatingsystem. Will need three different tests to correspond with the code in params.

```ruby
describe 'epel class' do
  context 'when osfamily is RedHat and operatingsystem is Fedora' do
    ...
  end
  context 'when osfamily is RedHat and operatingsystem is Fedora' do
    ...
  end
  context 'when osfamily is RedHat and operatingsystem is not Fedora' do
    ...
  end
  context 'when osfamily is not RedHat' do
    ...
  end
end
```

to set the facts, it would look like this

```ruby
context 'whatever' do
  let(:facts) do
    {
      :osfamily        => 'RedHat',
      :operatingsystem => 'Fedora',
    }
end
```